scripts/valhalla_build_timezones

```diff
@@ -2,7 +2,7 @@
 
 error_exit() {
   echo "error: $1" 1>&2
-  exit $(pkg-config geos --modversion | grep -cvF "3.9")
+  #exit $(pkg-config geos --modversion | grep -cvF "3.9")
```

Comment:
this is unrelated to this pr and i can remove it but a previous, semi-recent pr, broke this for geos 3.9 users. i'll have a fix for it in the chinese postman pr